.local/bin/statusbar/sb-forecast

```diff
@@ -48,6 +48,4 @@ case $BLOCK_BUTTON in
 	6) "$TERMINAL" -e "$EDITOR" "$0" ;;
 esac
 
-checkforecast || getforecast
-
-showweather
+checkforecast && showweather || { ( getforecast && pkill -RTMIN+5 "${STATUSBAR:-dwmblocks}") & echo; }
```

Comment:
Maybe it would make sense to use -RTMIN+${1:-5} instead of just -RTMIN+5 so that the signal number could be passed as an argument in the dwmblocks configuration

Reply:
Yeah, that's a really good idea, especially considering otherwise you would have to standardize and hard-code the currencies used in `sb-price`.